windows/hid.c

```diff
@@ -360,25 +383,57 @@ HID_API_EXPORT const char* HID_API_CALL hid_version_str(void)
 
 int HID_API_EXPORT hid_init(void)
 {
-	register_global_error(NULL);
+    register_global_error(NULL);
+    if (!hid_hotplug_context.mutex) {
+        hid_hotplug_context.mutex = CreateMutexA(NULL, FALSE, NULL);
```

Comment:
I'd prefer native Unicode `CreateMutexW` version.

Reply:
Okay. But, theoretically, this doesn't matter, as we don't assign it any name anyway.